docs/_articles/en/v3.x/ci-cd-integration.md

```diff
@@ -19,7 +19,29 @@ Follow these CI/CD best practices
 * Use the `-o/--outfile=<name>.xml` flag to write your results to a file and produce a results artifact that can be used by the CI/CD tool.
 * Use the `-s/–severity-threshold` flag to cause a non-zero exit code when any violations meet or exceed the provided value. Many CI/CD tools require thresholds.
 
-## Accelerate Your CI/CD Integration
+## Accelerate Your CI/CD Integration#
+
+### Run Code Analyzer GitHub Action
+
+To accelerate your continuous integration/continuous (CI/CD) development, create a GitHub Action workflow that uses the `run-code-analyzer` GitHub Action. GitHub Action workflows provide opportunities to automate your entire pipeline, from building and testing to deployment. The `run-code-analyzer` GitHub Action scans your code for violations using Salesforce Code Analyzer, uploads the results as an artifact, and displays the results as a job summary. 
+
+If you’re using DevOps Center, you can use the `run-code-analyzer` GitHub Action as you promote changes, helping you identify and address issues earlier in your development pipeline.
+
+With the `run-code-analyzer` Action, you can customize:
+
+* Which Salesforce Code Analyzer command to run: run or run dfa
+* What arguments to pass with your scan
+* The name of the results artifact
+
+With `run-code-analyzer`, these outputs are included in the results file.
```

Comment:
This is technically incorrect. Your bullet points here should be referring to the output variables that are then available to the users the GitHub workflow. The results file (included in the job artifact) is separate from the output variables.

Reply:
@stephen-carter-at-sf Hmm. This text is from the G doc that you and John reviewed/approved, and that CX has subsequently approved. The best process is for all edits to be incorporated into the G doc drafts, then post-approval I create the PR as I did today, not to make substantive edits in the PR itself.

Reply:
Unfortunately, let's go back to the G doc. If you could add your revisions to the GitHub Actions section so I can understand what updated you'd like me to make, that'd be great. Thank you!

https://docs.google.com/document/d/1-rzeL9ExghYZfH1Sewde381356n4jvKF4AFn6F2QFQM/edit?usp=sharing

Reply:
Done.